scripts/scenario_08_atlantis.lua

```diff
@@ -977,15 +977,19 @@ function update(delta)
 
     -- If the player enters the Kraylor defense line, or engages a forward station, attack with all forces.
     for _, warp_jammer in ipairs(kraylor_defense_line) do
-        if distance(player, warp_jammer) < 6000 then
-            putKraylorDefenseLineOnFullOffense()
-        end
+    	if warp_jammer ~= nil and warp_jammer:isValid() then
+			if distance(player, warp_jammer) < 6000 then
```

Comment:
Tabs vs spaces

Reply:
Uh, I used tabs. Are you saying I should use spaces?

Reply:
I think I sorted it